spec/cloud.go

```diff
@@ -122,7 +122,7 @@ var CloudGeneratorSuggester *cloudGeneratorSuggester
 
 func init() {
 	ec2BaseURL, _ = url.Parse("http://169.254.169.254/latest/meta-data")
-	gceMetaURL, _ = url.Parse("http://metadata.google.internal./computeMetadata/v1/?recursive=true")
```

Comment:
Without `/?recursive=true`, we cannot get GCE Metadata in `requestGCEMeta`.

```
astj@instance-1:~$ curl -H 'Metadata-Flavor: Google' http://metadata.google.internal./computeMetadata/v1
/computeMetadata/v1/
astj@instance-1:~$ curl -H 'Metadata-Flavor: Google' 'http://metadata.google.internal./computeMetadata/v1/?recursive=true'
{"instance":{"attributes":{"ssh-keys": ... (snip)
```

Reply:
Oops, It's my fault. I'm going add `?recursive=true` in requestGCEMeta.